package.json

```diff
@@ -2,7 +2,7 @@
 	"name": "esp-launchpad",
 	"dependencies": {
 		"crypto-js": "^4.1.1",
-		"esptool-js": "^0.2.1",
+		"esptool-js": "^0.2.2",
```

Comment:
Should this be 0.3.0 to point to latest?

Reply:
There are some API changes in 0.3.0 (https://github.com/espressif/esptool-js/pull/101) and I didn't have time to update esp-launchpad for those changes.

0.2.2, on the other hand, has only updated the flasher stub, which seems to fix one of the reported issues.